webrisk/snippets/compute_threatlist_diff.py

```diff
@@ -0,0 +1,81 @@
+#  Copyright 2022 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+# [START webrisk_compute_threatlist_diff]
+from google.cloud import webrisk_v1
+
+
+def compute_threatlist_diff(threat_type: webrisk_v1.ThreatType, version_token: bytes, max_diff_entries: int,
+                            max_database_entries: int, compression_type: webrisk_v1.CompressionType) -> None:
+    """
+    Gets the most recent threat list diffs. These diffs should be applied to a local database of
+    hashes to keep it up-to-date.
+
+    If the local database is empty or excessively out-of-date,
+    a complete snapshot of the database will be returned. This Method only updates a
+    single ThreatList at a time. To update multiple ThreatList databases, this method needs to be
+    called once for each list.
+
+    Args:
+        threat_type: The threat list to update. Only a single ThreatType should be specified per request.
+                     threat_type = webrisk_v1.ThreatType.MALWARE
```

Comment:
The comments do not need to be lined up, but rather spaced more tightly to conserve horizontal space. See below examples.
```suggestion
        threat_type: The threat list to update. Only a single ThreatType should be specified
            per request.
            threat_type = webrisk_v1.ThreatType.MALWARE
```